x-pack/plugins/security_solution/public/flyout/right/components/header_title.tsx

```diff
@@ -86,7 +86,7 @@ export const HeaderTitle: VFC<HeaderTitleProps> = memo(({ flyoutIsExpandable })
       )}
       <EuiSpacer size="s" />
       <EuiTitle size="s">
-        <h4 data-test-subj={FLYOUT_HEADER_TITLE_TEST_ID}>
+        <h2 data-test-subj={FLYOUT_HEADER_TITLE_TEST_ID}>
```

Comment:
shouldn't this be `h3`? Looking at the Title section on [EUI](https://eui.elastic.co/#/display/title), they're using `h3` with `<EuiTitle size="s">`